### DISCUSSION TITLE
How to improve zonal fault tolerance when deploying etcd to two zones.

### DISCUSSION THREAD

**Discussion by awx-fuyuanchu** [etcd-io/etcd#17769] [Q&A] [answered]
Currently, the IaaS Cloud only provides two AZs available in a region. Is there any best practice about the etcd configuration to improve the zonal fault tolerance?

**Comment by jmhbnz** [comment, marked as answer]
Hey @awx-fuyuanchu - Thanks for your question, please refer to:
- https://etcd.io/docs/v3.5/faq/#why-an-odd-number-of-cluster-members
- https://etcd.io/docs/v3.5/faq/#what-is-failure-tolerance

If you absolutely must operate across only two AZ's then I would consider running two clusters if fault tolerance of an AZ is a requirement and only two AZ's are available.
